_maps/WIP/Sulaco_bottom.dmm

```diff
@@ -4883,8 +4882,20 @@
 /turf/open/floor/plating,
 /area/sulaco/telecomms)
 "kI" = (
-/turf/open/floor/engine/vacuum,
-/area/sulaco/telecomms)
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8"
+	},
+/obj/machinery/atmospherics/pipe/simple/yellow/hidden{
```

Comment:
Are these map changes intended?

Reply:
Probably, I had to remove a couple PDA related items that LaKiller didn't from the map. And the .dmm probably reorganized itself, as result.